firedrake/mesh.py

```diff
@@ -827,7 +848,8 @@ def __init__(self, mesh, layers):
         self._cell_numbering = mesh._cell_numbering
         self._entity_classes = mesh._entity_classes
         self._subsets = {}
-        self._ufl_cell = ufl.TensorProductCell(mesh.ufl_cell(), ufl.interval)
+        cell = ufl.TensorProductCell(mesh.ufl_cell(), ufl.interval)
+        self._ufl_mesh = ufl.Mesh(ufl.VectorElement("Lagrange", cell, 1, dim=cell.topological_dimension()))
```

Comment:
I guess an extruded point is an interval, so this will fall out OK.

Reply:
I haven't added tests for extrusion with points in #1638  yet - thanks for the heads up. If there are issues I'll revisit in #1638 .